components/messages/terminal-messages/file-content-block.tsx

```
@@ -38,10 +35,13 @@ export const FileContentBlockComponent: React.FC<
     <div
       className={`overflow-hidden rounded-lg border border-border ${index === 1 ? 'mb-3' : 'my-3'}`}
     >
-      <div className="flex items-center justify-between border-b border-border bg-muted px-4 py-2">
+      <button
+        className="flex items-center justify-between border-b border-border bg-muted px-4 py-2 hover:opacity-50 cursor-pointer"
+        onClick={handleActionClick}
+      >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)